TA_dataset/README/ta_dataset_settings.conf.spec

```diff
@@ -1,5 +1,9 @@
 [account]
 url = <string>
+tenant = <bool>
+account_ids = <string>
+an_fir_part = <string>
```

Comment:
Why is the field name so cryptic? would it be possible to call it eg. `authn_token_part_one` ?

Reply:
because there is limitations on the splunk encrypted keys. in order to achieve that, used this type of names

Reply:
what is the limitation?

Reply:
all encrypted values are saved in single instance of Storage Manager which does not allow data <256 chars

Reply:
Yes I understand that from description, my point is not the value of this variable but variable name, which I found bit difficult to decrypt `an_fir_part `, I suggest eg. `authn_token_part_one `